monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -736,5 +736,76 @@
         },
       ],
     },
+    {
+      name: 'rbdmirror',
+      rules: [
+        {
+          alert: 'CephRBDMirrorConnectionLost',
+          'for': '1m',
+          expr: 'ceph_rbd_mirror_connection_health != 0',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: 'RBD Mirror connection with remote cluster is not working properly',
+            description: "RBD Mirror connection is lost, but mirror daemon is up",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesPerDaemonHigh',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status {daemon=xx) >= 100| |high image lag time|possible resource constraints or high data transfers|(rbd_mirror_image_status\{image=X, pool=Y, last_sync_duration)} == 1) > rbd_mirror_image_schedule{image=X, pool=Y, interval}',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status !=1) > 0',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "There are images where the replication process have failed.",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
```

Comment:
Both alerts have same name compare to line 765, If `expr` is different it should be different Is it?

Reply:
Changed